TestFile.java

```diff
@@ -0,0 +1,5 @@
+package com.remark;
```

Comment:
I would prefer to put this file into a directory, maybe `__fixtures__`? Also, I would prefer to just use a simple `js` file like shown in the README. The file name could also be the same.